### PR TITLE
CCTRI-768: Add Sightings and Targets mappings.

### PR DESCRIPTION
**Pull request by AlinaBlahun**

### CODE REVIEW

File: api/mappings.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        def sighting(asset, artifact):
+            return {
+                'id': f'transient:{uuid4()}',
+                'type': 'sighting',
+                'schema_version': '1.0.16',
+                'confidence': 'High',
+                'count': len(assets),
+                'source': 'Chronicle',
+                'source_uri': data['uri'][0],
+                'internal': 'True',
```
Comment (gdavoian) [inline api/mappings.py]: This field has to be a boolean, not a string :)

File: tests/unit/api/test_enrich.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from http import HTTPStatus
 
+from mock import patch
```
Comment (gdavoian) [inline tests/unit/api/test_enrich.py]: `from unittest.mock import patch`.

File: api/mappings.py (hunk 1 above)
Comment (AlinaBlahun) [reply api/mappings.py]: Done. Thanks a lot!

File: tests/unit/api/test_enrich.py (hunk 2 above)
Comment (AlinaBlahun) [reply tests/unit/api/test_enrich.py]: Done. Thanks!

**Pull request merged**